src/capitalsFirst.js

```diff
@@ -19,6 +19,19 @@
  */
 function capitalsFirst(str) {
   // write code here
-}
+  let arr = str.split(' ');
+  let arrUpperCase = [];
+  let arrLowerCase = [];
```

Comment:
you don\`t need let if you don\`t reassign variable(all three above)

Reply:
So i should use const?

Reply:
Sorry, I did't read all )